Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -495,6 +495,8 @@ The list of available options is reported in the :ref:`table <tab:cmakevar>` bel
    +------------------------------+-------------------------------------------------+-------------------------+-----------------------+
    | AMReX_BOUND_CHECK            |  Enable bound checking in Array4 class          | NO                      | YES, NO               |
    +------------------------------+-------------------------------------------------+-------------------------+-----------------------+
+   | AMREX_EXPORT_DYNAMIC         |  Something backtrace on macOS                   | NO (unless Darwin)      | YES, NO               |
```

Comment:
Fixme

Reply:
```
   | AMReX_EXPORT_DYNAMIC         |  Enable backtrace on macOS                   |  NO (YES, if Darwin OS)     | YES, NO               | 
```
It's hard to succinctly describe that variable default option so I thought I would at least put up another version to think about.

Reply:
Yes, I was wondering if there is more to it, thus I kept a placeholder that would raise suspicion for PR review :)